lib/OpenQA/WebAPI.pm

```diff
@@ -216,9 +216,9 @@ sub startup {
 
     # Favicon
     $r->get('/favicon.ico' => sub { my $c = shift; $c->render_static('favicon.ico') });
+    $r->get('/index' => [format => ['html', 'json']])->to('main#index');
```

Comment:
Ok. With my version, if you enter `index.txt` you get back to `index`. With your version you get 404 in this case.